### PR TITLE
Added release documentation

### LINKED ISSUE #591: Create a release document
We should document the release process.

For reference - https://github.com/Particular/ServicePulse/blob/master/docs/releasing.md

### PR DESCRIPTION
**Pull request by distantcam**
Resolves #591

### CODE REVIEW

**Comment by distantcam** [comment]
@HEskandari Can you review this please.

File: docs/releasing.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+Once you have your change merged into `master` (AVOID BUNDLING CHANGES, RELEASE OFTEN INSTEAD) and the [build is green](https://builds.particular.net/project.html?projectId=ServiceInsight) it is time to "Promote" the master build to "Deploy".
+![Promote](Promote.PNG)
+![Promote_dialog](Promote_dialog.PNG)
+
+### 3. Deploy using OctopusDeploy
+You are now ready to go to [OctopusDeploy](http://deploy.particular.net/app#/projects/serviceinsight).
+You should have a new "Staging" build waiting to be released.  
+![Octopus_staging](Octopus_staging.PNG)
+
+First step is to proof read the release notes that were auto generated (it is best to use an outsider for this job).
```
Comment (HEskandari) [inline docs/releasing.md]: What does 'outsider' mean here?

File: docs/releasing.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+## ServiceInsight releasing procedure
+
+### 1. Merge change
```
Comment (HEskandari) [inline docs/releasing.md]: Shouldn't we ensure that the issues are assigned to a milestone first?

File: docs/releasing.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+**NOTE 2**: The link at the end needs to be updated to:
+```
+You can download this release from our [website](http://particular.net/downloads).
+```
+
+Once you happy approve them:  
+![looks good](looks%20good.PNG)
+
+And then all is left to do is "Promote to Production".  
+![promote to prod](promote%20to%20prod.PNG)
```
Comment (HEskandari) [inline docs/releasing.md]: Do we include the announcement as a part of the release process?

**Comment by HEskandari** [comment]
@distantcam I have made some comments, but overall I like it.

File: docs/releasing.md (hunk 2 above)
Comment (distantcam) [inline docs/releasing.md]: Yes. I'll add some notes to the deploy section.

File: docs/releasing.md (hunk 3 above)
Comment (distantcam) [inline docs/releasing.md]: I'd say no at this point. I'm not sure if this should just be a technical document vs what the company procedures are.

**Pull request merged**